exasol/ai/mcp/server/mcp_server.py

```diff
@@ -272,15 +274,18 @@ def describe_table(
                     C.COLUMN_TYPE AS "{conf.type_field}",
                     C.COLUMN_COMMENT AS "{conf.comment_field}",
                     NVL2(P.COLUMN_NAME, TRUE, FALSE) AS "{conf.primary_key_field}",
-                    NVL2(F.COLUMN_NAME, TRUE, FALSE) AS "{conf.foreign_key_field}"
+                    NVL2(F.COLUMN_NAME, CONCAT(
```

Comment:
If I recall correctly, foreign keys can have names and can use multiple columns as key